tests/performance/apdex-score/test/page-objects/load.page.js

```diff
@@ -1,19 +1,23 @@
 const { $ } = require('@wdio/globals');
 const Page = require('./page');
 
-class LoadPage extends Page {
-   
-  get inputInstanceUrl () {
+class LoadPage extends Page {  
+  
+  get btnCustom() {
+    return $('//*[@text="Custom"]');
```

Comment:
It is better to specify the type of element instead of using `*` I guess that might be `android.widget.Button`, not sure, can you please check?

Reply:
it's `android.widget.TextView` thanks